Add details about organization roles (#2825)
wandb/docs: platform/hosting/iam/access-management/manage-organization.mdx

| Viewer (Enterprise-only feature) | A view-only user of your organization, invited by an organization admin. A viewer only has read access to the organization and the underlying teams that they are a member of. |
| Billing only (Multi-tenant Cloud) | A billing-focused role for Multi-tenant Cloud organizations. You can assign this role only to the organization's billing user. Billing-only users can manage billing workflows, such as subscriptions, invoices, and payment methods, but they can't invite users, manage teams, or perform non-billing admin actions. Billing-only users are also removed from all organization teams and always have **No access** for both Models and Weave. |
| Custom Roles (Enterprise-only feature) | Custom roles allow organization admins to compose new roles by inheriting from the preceding **Viewer** or **Member** organization roles, and adding additional permissions to achieve fine-grained access control. Team admins can then assign any of those custom roles to users in their respective teams. See also [Add and manage custom roles](#add-and-manage-custom-roles). |

### Organization role permissions

> Organization roles control administrative actions at the organization scope. They're separate from [team roles](#assign-or-update-a-team-members-role), which control what a user can do inside a specific team, and from [Models seat and Weave access](#assign-or-update-a-users-access), which control product access and billing.

Reviewer: Do we want to also include registry roles? E.g. "They are separate from team roles or registry roles..."

Author: Good idea. Done, with a bit of restructuring.

> Organization roles control administrative actions at the organization scope. They're separate from [team roles](#assign-or-update-a-team-members-role), which control what a user can do inside a specific team, and from [Models seat and Weave access](#assign-or-update-a-users-access), which control product access and billing.

Author: ```suggestion
Organization roles control administrative actions at the organization scope. They're separate from these other sets of roles:

- [Team roles](#assign-or-update-a-team-members-role)  control what a user can do inside a specific team.
- [Registry roles](https://docs.wandb.ai/models/registry/configure_registry#details-about-registry-roles) control what a user can do in a given registry.
- [Models seats and Weave access](#assign-or-update-a-users-access) control product access and billing.
```

| Custom Roles (Enterprise-only feature) | Custom roles allow organization admins to compose new roles by inheriting from the preceding **Viewer** or **Member** organization roles, and adding additional permissions to achieve fine-grained access control. Team admins can then assign any of those custom roles to users in their respective teams. See also [Add and manage custom roles](#add-and-manage-custom-roles). |

### Organization role permissions

Organization roles control administrative actions at the organization scope. They're separate from these other sets of roles:

- [Team roles](#assign-or-update-a-team-members-role)  control what a user can do inside a specific team.
- [Registry roles](https://docs.wandb.ai/models/registry/configure_registry#details-about-registry-roles) control what a user can do in a given registry.

> - [Models seats and Weave access](#assign-or-update-a-users-access) control product access and billing.

Reviewer 2: nit: remove "and billing"

> - [Models seats and Weave access](#assign-or-update-a-users-access) control product access and billing.

Author: ```suggestion
- [Models seats and Weave access](#assign-or-update-a-users-access) control product access.
```

The following tables summarize what **Member** and **Admin** users can do at the organization level. For team-scoped actions such as logging runs or editing reports, see [Team roles and permissions](/platform/app/settings-page/teams#team-roles-and-permissions). For registry-scoped actions, see [Registry role permissions](/models/registry/configure_registry#role-permissions).

> <Note>

Reviewer 2: is this the right place to mention this? 

Custom roles and viewer roles are team level, not org level.

<Note>
On Self-Managed deployments, instance admins can optionally enable team creation for non-admin users. This option is managed at the instance level, not in organization settings. If you don't know whether your instance has this option enabled, contact your instance admin or W&B support.
</Note>

Organization admins are assigned the **Admin** role in [registries](/models/registry/configure_registry#role-permissions) that are visible at the organization level.

#### Organization settings

| Permission                                                   | Permission group | Member | Admin |
| ------------------------------------------------------------ | --------------- | :----: | :---: |
| Change the organization name (Multi-tenant Cloud)            | Admin           |        |   X   |
| Configure domain capture and SSO auto-provisioning           | Admin           |        |   X   |
| Enforce organization-wide [privacy settings](/platform/hosting/privacy-settings#enforce-privacy-settings-for-all-teams) | Admin |        |   X   |
| Create, edit, and delete custom roles (Enterprise)           | Admin           |        |   X   |

> | Configure organization alert thresholds                      | Admin           |        |   X   |

Reviewer 2: is this referencing automatons or slack alerts? 

Or is it a duplicate of billing alerts, which is is documented below as "configure usage alerts".

> | Configure organization alert thresholds                      | Admin           |        |   X   |

Author: ```suggestion
```

| Connect external storage buckets at the organization level (Enterprise) | Admin |        |   X   |

#### Billing

| Permission                                                   | Permission group | Member | Admin |
| ------------------------------------------------------------ | --------------- | :----: | :---: |
| Assign the billing admin                                     | Admin           |        |   X   |
| Manage subscriptions, payment methods, and plan upgrades     | Billing         |        |   X   |
| Cancel subscriptions                                         | Billing         |        |   X   |
| Configure usage alerts                                       | Billing         |        |   X   |

> Users designated as **billing admins** can perform billing actions, including configuring usage alerts, even if their organization role is **Member**. Billing admins can't perform other organization admin actions such as inviting users or creating teams.

Reviewer 2: unless they are an org admin

> Users designated as **billing admins** can perform billing actions, including configuring usage alerts, even if their organization role is **Member**. Billing admins can't perform other organization admin actions such as inviting users or creating teams.

Author: ```suggestion
Users designated as **billing admins** can perform billing actions, including configuring usage alerts, even if their organization role is **Member**. Unless they are also an organization admin, a billing admin can't perform other organization admin actions such as inviting users or creating teams.
```

On Multi-tenant Cloud, you can optionally assign the billing admin the **Billing only** organization role. This role is restricted to the billing user, removes team membership, and enforces **No access** for both Models and Weave while preserving billing administration capabilities.

#### Security and automation

| Permission                                                   | Permission group | Member | Admin |
| ------------------------------------------------------------ | --------------- | :----: | :---: |
| Authenticate to the [SCIM API](/platform/hosting/iam/scim)   | Admin           |        |   X   |
| Create and manage organization-scoped service accounts        | Admin           |        |   X   |
| View and manage all organization API keys                    | Admin           |        |   X   |

> | Manage organization user secrets (Dedicated Cloud and Self-Managed) | Admin     |        |   X   |

Reviewer 2: what does this refer to? does this exist?

> | Manage organization user secrets (Dedicated Cloud and Self-Managed) | Admin     |        |   X   |

Author: ```suggestion
```

</Note>

#### User management

| Permission                                                   | Permission group | Member | Admin |
| ------------------------------------------------------------ | --------------- | :----: | :---: |
| View the organization user list                              | Read            |   X    |   X   |
| Invite users to the organization                             | Admin           |        |   X   |
| Remove users from the organization                           | Admin           |        |   X   |
| Assign or change a user's organization role                  | Admin           |        |   X   |
| Assign or change a user's Models seat or Weave access        | Admin           |        |   X   |
| Export the organization user list as CSV                     | Admin           |        |   X   |
| View organization activity and usage trends                  | Read            |   X    |   X   |

> In organizations marked as hidden, non-admin members can view profiles only for themselves or for users who share a team with them.

Reviewer 2: Do you mean: 
In organizations with :Enforce team visibility restrictions: enabled?  https://weightsandbiases.slack.com/archives/C069VUZS152/p1782318870167949

if not, what does "hidden" refer to? how can an admin mark an organization as hidden?

Author: Updated

#### Teams

| Permission                                                   | Permission group | Member | Admin |
| ------------------------------------------------------------ | --------------- | :----: | :---: |
| Create teams in the organization                             | Admin           |        |   X   |
| Join teams when invited or through domain capture            | Read            |   X    |   X   |
| View team membership for teams the user belongs to           | Read            |   X    |   X   |
| View team membership for teams the user doesn't belong to    | Read            |        |   X   |
| Invite users to a team without being a member of that team   | Admin           |        |   X   |

> | Add themselves to a team as a member or viewer               | Admin           |        |   X   |

Reviewer 2: Can an org admin add themselves to a team as a team admin? This implies they cannot, which seems odd to me.

Author: After looking at the table output, I think what it means is that an org admin can't add themselves to a team as a member, becausse org admin effectively overrides it. WDYT?

<Note>

> On Self-Managed deployments, non-admin users might be allowed to create teams when your instance administrator enables that option.

Reviewer 2: Is there documentation on this option? (if yes, can we link to it). If no, can we be more detailed in how one would identify if this option was enabled on the instance (or not)

| Change the organization name (Multi-tenant Cloud)            | Admin           |        |   X   |
| Configure domain capture and SSO auto-provisioning           | Admin           |        |   X   |
| Enforce organization-wide [privacy settings](/platform/hosting/privacy-settings#enforce-privacy-settings-for-all-teams) | Admin |        |   X   |
| Create, edit, and delete custom roles (Enterprise)           | Admin           |        |   X   |
| Configure organization alert thresholds                      | Admin           |        |   X   |
| Connect external storage buckets at the organization level (Enterprise) | Admin |        |   X   |

#### Billing

| Permission                                                   | Permission group | Member | Admin |
| ------------------------------------------------------------ | --------------- | :----: | :---: |
| Assign the billing admin                                     | Admin           |        |   X   |
| Manage subscriptions, payment methods, and plan upgrades     | Billing         |        |   X   |
| Cancel subscriptions                                         | Billing         |        |   X   |

> | View billing alert thresholds                                | Read            |        |   X   |

Reviewer 2: I would assume that Admins could also edit billing alert thresholds. 

side note: is this the same word choice that the usage alert documentation uses? Should it be "usage alert thresholds"
